src/test/java/com/purbon/kafka/topology/integration/SaslPlaintextKafkaContainer.java

```diff
@@ -0,0 +1,142 @@
+package com.purbon.kafka.topology.integration;
+
+import com.github.dockerjava.api.command.ExecCreateCmdResponse;
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import org.apache.kafka.clients.CommonClientConfigs;
+import org.apache.kafka.clients.admin.Admin;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.images.builder.Transferable;
+import org.testcontainers.utility.DockerImageName;
+
+/** @author <a href="mailto:shh@thathost.com">Sverre H. Huseby</a> */
+final class SaslPlaintextKafkaContainer extends GenericContainer<SaslPlaintextKafkaContainer> {
+
+  private static final DockerImageName DEFAULT_IMAGE =
+      DockerImageName.parse("confluentinc/cp-kafka").withTag("6.0.0");
```

Comment:
while I love using the latest for tests. The current tests are using a much lower version. 

Would you align the version across all tests? In the long run, I would love to have this as a parameter so we can run tests across multiple CP versions.

Reply:
I downgraded to 5.5.1, as used in the RBAC docker-compose. Had to use another authorizer class.